docs/roadmap.md

```diff
@@ -13,22 +13,22 @@ so we have a lot of things on our plate!
 Due to the nature of the product, our roadmap is doomed to evolve
 often according to the technologies around it.
 
-Here are examples:
+Here are a few examples:
```

Comment:
I added "a few" here as it felt a bit better, this is not really necessary though